src/core/src/main/java/org/apache/jmeter/util/JMeterUtils.java

```diff
@@ -1282,11 +1282,7 @@ public static void setupXStreamSecurityPolicy(XStream xstream) {
      * @return variable name for index following JMeter convention
      */
     public static String formatJMeterExportedVariableName(String elementName) {
-        StringBuilder builder = new StringBuilder(
-                JMETER_VARS_PREFIX.length()+elementName.length());
```

Comment:
Technically speaking, this was slightly more efficient in Java 8 than the suggested `+`.

However I don't know if that optimization matters.

Reply:
I'm not sure it matters much, and in a very simple benchmark:
```
Benchmark               Mode  Cnt        Score        Error  Units
MyBenchmark.newMethod  thrpt    6  4659748.533 ± 195139.536  ops/s
MyBenchmark.original   thrpt    6  4771658.266 ±  31374.782  ops/s
```